soccer/MainWindow.cpp

```diff
@@ -689,8 +727,8 @@ void MainWindow::updateViews() {
             }
 
             // check for kicker error code
-            bool kickerFault =
-                rx.has_kicker_status() && (rx.kicker_status() & 0x80);
+            bool kickerFault = rx.has_kicker_status() &&
+                               !(rx.kicker_status() & Kicker_Enabled);
```

Comment:
Yes, there was definitely a corresponding change in firmware that we made when cleaning up some kicker status stuff. This is correct